src/app/components/home/thumbnail/preview.component.ts

```diff
@@ -29,7 +29,7 @@ export class PreviewComponent implements OnInit, OnDestroy {
   @Input() title: string;
 
   hover: boolean;
-  currentlyShowing = 1;
+  currentlyShowing = 0;
```

Comment:
Do you prefer the first screenshot?

I've found that usually the screenshot from the 5% mark in the video often has useless content (in a cartoon it's often the opening credits - at worst rendering all screenshots from a cartoon series the same).

Reply:
Ha, it feels a little funny to me not to start at the first image, but you have a good point! 👍